views/applications/all.html

```diff
@@ -0,0 +1,49 @@
+{% extends "_donut.html" %}
+
+{% block pageHead %}
+  {{ super() }}
+{% endblock %}
+
+{% block header %}
+  {{ super() }}
+{% endblock %}
+
+{% block main %}
+  <div class="introduction">
+    <h1 class="introduction--h1">{{structure.name}}</h1>
+    <p class="introduction--p">{{structure.description}}</p>
+    <a href="/${config.base}/${config.actions.add}" class="introduction--button">{{config.actions.add|capitalize}} {{config.base}}</a>
+  </div>
+
+  {% if applications %}
+  <table class="base--table">
+    <thead class="base--thead">
+      <tr class="base--tr">
+        <th class="base--th">Application</th>
+        <th class="base--th">Created</th>
+        <th class="base--th">Live</th>
+        <th class="base--th">Updated</th>
+        <th class="base--th">Sunset</th>
+        <th class="base--th">Edit</th>
+      </tr>
+    </thead>
+    <tbody class="base--tbody">
+    {% for a in applications %}
+      <tr class="base--tr">
+        <td data-th="Application" class="base--td">{{a.name}}</td>
+        <td data-th="Created" class="base--td">{{a.created}}</td>
+        <td data-th="Live" class="base--td">{{a['live-endpoint']}}</td>
```

Comment:
@scottnath I'm not moving the goalpost, this is the conversation we had Wednesday and Thursday about this view while you were looking for advice for implementing. For _just_ the views, this doesn't increase scope in a significant way. This is _just_ the views. It may wind up increasing the scope of the the controller story, but not by much (the audit trail already in the DB will provide that information), grabbing the latest entry and seeing if it passed or not, and then passing to the view if it indeed passed or failed with an if statement accordingly